lib/ex_machina/sequence.ex

```diff
@@ -1,6 +1,6 @@
 defmodule ExMachina.Sequence do
   def start_link do
-    Agent.start_link(fn -> HashDict.new end, name: __MODULE__)
+    Agent.start_link(fn -> %{} end, name: __MODULE__)
```

Comment:
How about using [`Map.new/0`](http://elixir-lang.org/docs/stable/elixir/Map.html#new/0) to keep similar level of abstraction?